pyproject.toml

```diff
@@ -88,23 +88,29 @@ Documentation = "https://papis.readthedocs.io"
 Repository = "https://github.com/papis/papis"
 
 [project.optional-dependencies]
-develop = [
+# Dev environment -- install if you're working on papis
+develop = ["papis[style,test]"]
+# Style enforcement
+style = [
```

Comment:
If we're separating these, I would vote for `lint = [flake8 + plugins]`, `typing = [mypy + type-stubs]` and `test = [pytest, pytest-cov]`. What do you think?

Reply:
Sounds good to me, making that change. Should I also have `develop` pull in `lsp`? Otherwise, if it's just a dev recommendation, I'd recommend instead documenting its usefulness in `CONTRIBUTING.md` and perhaps in the comment here.

I'm not seeing the comment on suggested names for `plugin`/`plugin_test`. Would prefixing them with `external_` help?
(In fact, it might be a good idea to instead split off those two modules into some kind of `papis-internals-utils` package -- that'd clarify the weak ties between it and `papis`. Not sure if that's too much code churn though)

Reply:
> I'm not seeing the comment on suggested names for plugin/plugin_test. Would prefixing them with external_ help?

I answered your other questions above: https://github.com/papis/papis/pull/985#issuecomment-2737485914. Is that what you were looking for?

>  (In fact, it might be a good idea to instead split off those two modules into some kind of papis-internals-utils package -- that'd clarify the weak ties between it and papis. Not sure if that's too much code churn though)

Yeah, I can't see that being too helpful beyond just being very loud that they're optional. Plus there's a bit of a circular dependency there.